caddyhttp/caddyhttp_test.go

```diff
@@ -25,7 +25,7 @@ import (
 // ensure that the standard plugins are in fact plugged in
 // and registered properly; this is a quick/naive way to do it.
 func TestStandardPlugins(t *testing.T) {
-	numStandardPlugins := 31 // importing caddyhttp plugs in this many plugins
+	numStandardPlugins := 34 // importing caddyhttp plugs in this many plugins
 	s := caddy.DescribePlugins()
```

Comment:
We wrote (I think) only one plug in, so we should have incremented numStandardPlugins to 32. But that didn't pass all the test cases. Ultimately we changed it to 34, which is the number it expected, and it passed. We're not usre if this was a valid fix, or if it was just symptomatic of a larger problem.